taxi/views.py

```diff
@@ -1,16 +1,18 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 from django.urls import reverse_lazy
-from django.views import generic
+from django.views import generic, View
 from django.contrib.auth.mixins import LoginRequiredMixin
-
+from django.shortcuts import get_object_or_404, redirect
+from django.contrib.auth import get_user_model
+from taxi.forms import DriverCreationForm
 from .models import Driver, Car, Manufacturer
```

Comment:
According to the checklist, you should use `get_user_model()` instead of `Driver`. Please, replace `Driver` with `get_user_model()`.